### PR TITLE
[BD-46] feat: Added fallback props to Card component

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

- add `fallbackSrc` prop to the `Card.ImageCap` component, use this `url` for image if image from the src prop does not load;
- add `fallbackLogoSrc` prop to Card.ImageCap component, use this `url` for logo if image from the `logoSrc` prop does not load;
- add tests for these changes and examples on the component's page;
- fixed key error in `ExamplePropsForm.jsx`.

### Deploy Preview

[Cards component](https://deploy-preview-1682--paragon-openedx.netlify.app/components/card/#with-fallback-image)

## Merge Checklist

* [x] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [x] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [x] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [x] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [x] Were your changes tested in the `example` app?
* [x] Is there adequate test coverage for your changes?
* [x] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [x] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [x] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [x] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!


When this pull request is ready, tag your edX technical lead.

File: www/src/components/exampleComponents/ExamplePropsForm.jsx [hunk 1]
```diff
@@ -2,67 +2,74 @@ import React from 'react';
...
-    {inputs.map(input => {
-      if (input.options) {
+function ExamplePropsForm({ inputs }) {
+  return (
+    <div className="pgn-doc__example-props-form">
+      <h4>Props panel</h4>
+      {inputs.map(input => {
+        if (input.options) {
+          return (
+            <Form.Group key={JSON.stringify(input)}>
```
Comment (PKulkoRaccoonGang) [inline www/src/components/exampleComponents/ExamplePropsForm.jsx:12]: added `key`

File: www/src/components/exampleComponents/ExamplePropsForm.jsx [hunk 2]
```diff
@@ -2,67 +2,74 @@ import React from 'react';
...
+              </Form.Label>
+              <Form.RadioSet
+                isInline
+                name={input.name}
+                onChange={(e) => input.setValue(e.target.value)}
+                value={input.value}
+              >
+                {input.options.map(option => (
+                  <Form.Radio
+                    key={option}
```
Comment (PKulkoRaccoonGang) [inline www/src/components/exampleComponents/ExamplePropsForm.jsx:24]: added `key`

**Review by adamstankiewicz — APPROVED** [review]
Nice! LGTM.

**Comment by viktorrusakov** [comment]
@adamstankiewicz true, added new tests and fixed linting issues, should be good to go now!

File: src/Card/tests/CardImageCap.test.jsx [hunk 3]
```diff
@@ -38,45 +39,65 @@ describe('<CardImageCap />', () => {
...
+        logoSrc="http://logo.image/"
+        fallbackLogoSrc="http://logo.image.fallback/"
+        logoAlt="Logo alt text"
       />,
     );
-    expect(wrapper.exists('.pgn__card-image-cap-loader')).toBe(true);
-    expect(wrapper.props().isLoading).toBe(true);
+    const srcImg = screen.getByAltText('Src alt text');
+    expect(srcImg.src).toEqual('http://src.image/');
+    fireEvent.error(srcImg);
```
Comment (adamstankiewicz) [inline src/Card/tests/CardImageCap.test.jsx:95]: TIL :) `fireEvent.error`

File: www/src/components/exampleComponents/ExamplePropsForm.jsx [hunk 4]
```diff
@@ -19,15 +19,20 @@ const ExamplePropsForm = ({ inputs }) => (
               value={input.value}
             >
               {input.options.map(option => (
-                <Form.Radio value={option.value || option}>{option.name || option.value || option}</Form.Radio>
+                <Form.Radio
+                  value={option.value || option}
+                  key={option.value || option}
```
Comment (adamstankiewicz) [inline www/src/components/exampleComponents/ExamplePropsForm.jsx:24]: Nice cleanup!

**Review by adamstankiewicz — APPROVED** [review]

**Pull request merged**